### ISSUE TITLE
Conda install vs pip install in jupyter notebook

### ISSUE THREAD

**Issue by kc1** [jupyter/notebook#3672]
I'm working in windows using git-bash . I've created an environment called jupyter1. When I look at it inside pycharm I see:[![enter image description here][1]][1]

I've then installed a number of packages using pip. When I click on the round circle on the right I see the pip installed packages:

[![enter image description here][2]][2]

This has been working fine for me in the pycharm IDE. 

Recently I've started to work in jupyter notebook . After starting a notebook and clicking on the conda tab and selecting jupyter1, I see:

[![enter image description here][3]][3]

None of the pip packages are available. How do I make these available?

  [1]: https://i.stack.imgur.com/mKk9J.png
  [2]: https://i.stack.imgur.com/X1iUR.png
  [3]: https://i.stack.imgur.com/ljCmw.png

**Comment by takluyver** [comment]
That conda tab is something that conda adds - it's not in the notebook code.

**Comment by kc1** [comment]
Thank you, I have posted this question in the github conda repo as well. Do you have any thoughts on how to get access to the ackages that have been pip installed in the conda env?

**Comment by takluyver** [comment]
I'd assume that they're there if you import them, just not showing up in the list.

**Comment by bollwyvl** [comment]
Right, it's probably just showing the output of "conda list" which only
knows what got there via "conda install"... While written in python, conda
can install all kinds of things, (e.g. node packages, r packages),
including other package managers... But can't really tell what those
package managers might do after the fact.

**Comment by kc1** [comment]
OK guys. thank you.
